ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/cygwin/tasks/main.yml

```diff
@@ -55,6 +55,17 @@
     - xterm
   tags: cygwin
 
+- name: Download grep-2.27-2
```

Comment:
Would be good to add a comment as to why we are using this particular version and warn folks not to upgrade it.